www/includes/easyparliament/templates/html/ni/index.php

```diff
@@ -0,0 +1,111 @@
+    <div class="hero">
+        <div class="row">
+        <div class="hero__mp-search">
+            <div class="hero__mp-search__wrap">
+                <h1>Do your MLAs represent you?</h1>
```

Comment:
This presumably matches the old behaviour, but I see this is less fully functional than UK/Scotland in terms of showing you existing etc. – is that easier to fix now these things are more overlapping?